extensions/amp-date-countdown/validator-amp-date-countdown.protoascii

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-date-countdown
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-date-countdown"
+    allowed_versions: "0.1"
+    allowed_versions: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+
+tags: {  # <amp-date-countdown>
+  html_format: AMP
+  tag_name: "AMP-DATE-COUNTDOWN"
+  requires_extension: "amp-date-countdown"
+  attrs: {
+    name: "end-date"
+    mandatory_oneof: "['end-date', 'timestamp-ms', 'timestamp-seconds']"
+  }
+  attrs: {
+    name: "timestamp-ms"
+    mandatory_oneof: "['end-date', 'timestamp-ms', 'timestamp-seconds']"
+  }
+  attrs: {
+    name: "timestamp-seconds"
+    mandatory_oneof: "['end-date', 'timestamp-ms', 'timestamp-seconds']"
+  }
+  attrs: {
+    name: "offset-seconds"
```

Comment:
I believe as of a recent commits, attrs are required to be in alphabetical order, by validator_test.js.